.github/workflows/build.yml

```
@@ -5,6 +5,7 @@ on:
     branches-ignore:
       - 'master'
       - 'oldsite'
+  pull_request:
```

The build process relies on some GitHub API tokens in the repo secrets that PRs from forks don't usually have access to, but maybe [`pull_request_target`](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request_target) will work?

```suggestion
  pull_request_target:
```